src/app/settings/page.tsx

```diff
@@ -45,9 +54,12 @@ export default function Settings() {
               onChange={handleThemeChange}
             />
           </div>
+          {/* Notifications (WIP) */}
+          {/*
           <hr className={styles.lineSpliter}></hr>
           <h2 className={styles.SectionLabel}>Notifications</h2>
           <ToggleSwitch isOn={isToggled} handleToggle={handleToggle} />
+          */}
```

Comment:
Do we still need this? if not lets avoid dead code, feel free to put an urgent story in place

Reply:
I decided to leave the notification portion commented out just in case we may circle around to doing this in the future. looking at it now it isn't anything crazy so I can go ahead and delete